### PR TITLE
Update docs language and formatting

### LINKED ISSUE #120: Django type interface from docs doesn't work
Hi :)
it seems that the interface attribute in the code example from [Docs](https://github.com/strawberry-graphql/strawberry-graphql-django/blob/main/docs/references/resolvers.md) is not working.

Is there an alternative way to declare interfaces?

### PR DESCRIPTION
**Pull request by augustebaum**
Fixed some of the language for readability.

Also closes #120.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by bellini666** [comment]
Hi @augustebaum ,

The PR seems empty. It seems that you've done your changes in one commit and then reverted them in the other.

**Comment by augustebaum** [comment]
Thanks for the heads-up! Sorry for the inconvenience ^^'

File: docs/references/filters.md [hunk 1]
```diff
@@ -79,38 +79,19 @@ class FruitFilter:
         return queryset.exclude(name='banana')
 ```
 
-## Overriding the filter method
```
Comment (bellini666) [inline docs/references/filters.md:82]: Why was this section removed?

**Review by bellini666 — CHANGES_REQUESTED** [review]
Thanks for the awesome contribution! :)

Just made a comment of a section that might have been removed by mistake. Just fixing that and this should be ready to be merged

File: docs/references/filters.md (hunk 1 above)
Comment (augustebaum) [reply docs/references/filters.md:82]: I thought it was redundant with the previous section title; I don't mind leaving it in though (or we could remove that part from the previous section title)?

**Comment by bellini666** [comment]
Hey @augustebaum ,

Just giving you a ping in case you didn't see my previous comment. There's just a small issue to solve, looking forward to merge your PR :)

**Comment by augustebaum** [comment]
Hi, very sorry for the long wait! I think I understand your point -- I took a really long time to spot the difference between the two examples though, so I thought making a substantial change would maybe help. I'd love your feedback on that (and also if my example is valid code).

**Review by bellini666 — APPROVED** [review]

**Pull request merged**